pages/common/live-server.md

```diff
@@ -0,0 +1,19 @@
+# live-server
+
+> A simple development http server with live reload capability.
+
+- Serve an index.html file and reload on changes:
```

Comment:
So it only serves `index.html`, and not any other files?

Reply:
index.html is the default entry file, if other files are referenced, they will be served as well. Not sure how to succinctly put that... Happy to take suggestions.

Reply:
@sbrl - index.html is the standard file which a web server returns. And it is mentioned below how to configure it to return some other file.

Reply:
Ah, sorry - I don't think I was particularly clear in that comment 😅 

I'm try to get at the fact that to me, it implies that it serves a single file, instead of files from a directory with a default file.

Reply:
I see.  "Serve an index.html" exactly means "serve files from a directory with index.html being the default file". It was pretty clear to me. Let me know if you have some alternate suggestions.